jwt/algorithms.py

```diff
@@ -171,6 +199,7 @@ def sign(self, msg, key):
     def verify(self, msg, key, sig):
         return constant_time_compare(sig, self.sign(msg, key))
 
+
```

Comment:
It's possible our `flake8` in CI just started caring about these. If so, carry on. As long as flake8 is happy, I'm happy.